src/cmd/vpnUp.go

```diff
@@ -143,28 +141,16 @@ func vpnUpCmd() *cmdBuilder.Cmd {
 		})
 }
 
-func isVpnUp(ctx context.Context, uxBlocks uxBlock.UxBlocks, attempts int) bool {
-	p := []uxHelpers.Process{
-		{
-			F: func(ctx context.Context) error {
-				for i := 0; i < attempts; i++ {
-					err := nettools.Ping(ctx, vpnCheckAddress)
```

Comment:
I would keep the ping there.
Just first check if the WG interface even exists, and if it does try to ping once or twice with few second timeout using `context.WithTimeout()` (ideally so it doesn't go over 5 seconds total).

Reply:
We had disscusion with @jansaidl about it and reached conclusion that it's not really good idea. For example when you activate vpn through `vpn up` command and then disconnect using os network manager, tunnel interface is still up but your not connected to it, so `dig` and `ping` are stuck, which caueses error when you try to run `vpn up` again. Also your project with core logger does not have to exist anymore, if you ran `vpn up` and then deleted project, or you have connection issues with your ISP... This seems like better solution to check wheter your connection is active, but there could be `vpn status` command which would tell you wheter your zerops project is reachable.

Reply:
The issue I have with it is, that this doesn't tell you if the connections is "actually" up (working). Having a WG interface there doesn't mean it works, so you will be falsely saying the VPN is up when it isn't.

Reply:
It checks whether vpn is active, which it is when WG interface exits. I mean you can have problem with your internet connection and existsing zerops interface, which means active vpn.

Reply:
What I mean is that I have experienced issues with VPN where my internet worked with no issues, the interface also existed, yet id didn't work. Only way to fix it was to stop and start the VPN again. So in such case zcli would show "yeah, your VPN is fine", when it actually isn't.

But I do agree that logger container not existing is an issue and am not sure how to work around that. Maybe check if the project VPN was started for still exists?

Reply:
Valid. I would like it to actually check if you are able to SSH to any of the containers in the project the SSH was started up for, but lets create a ticket for that, as it's more of a nice to have for now.